### PR TITLE
init project add .gitignore and  build.gradle file

### PR DESCRIPTION
**Pull request by raulchoque-eng**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+#reports
+/target/*
+
+#Log4J
+/log/*
```
Comment (TheYisusGithub) [inline .gitignore]: Dear raul please put salt of the line in this site.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    // https://mvnrepository.com/artifact/io.cucumber/cucumber-testng
+    compile group: 'io.cucumber', name: 'cucumber-testng', version: '4.7.1'
+    // https://mvnrepository.com/artifact/io.cucumber/cucumber-java
+    compile group: 'io.cucumber', name: 'cucumber-java', version: '4.7.1'
+    compile group: 'net.masterthought', name: 'cucumber-reporting', version: '4.6.0'
+    // https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java
+    compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.141.59'
+    // https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager
+    compile group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '3.6.2'
```
Comment (TheYisusGithub) [inline build.gradle]: Raul we must be used maven repository.

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'GUIAutomation'
```
Comment (TheYisusGithub) [inline settings.gradle]: Don't forget to use Maven don't use Gradle, please.

File: .gitignore (hunk 1 above)
Comment (raulchoque-eng) [reply .gitignore]: thank you, this wrong was fixed

**Pull request merged**